tests/test_countgraph.py

```diff
@@ -1472,14 +1471,14 @@ def test_abund_dist_gz_bigcount_compressed_first():
     f_out.close()
     # load the compressed bigcount table
     try:
-        countgraph = khmer.load_countgraph(outfile)
+        countgraph = Countgraph.load(outfile)
     except OSError as err:
         assert 0, 'Should not produce OSError: ' + str(err)
 
     assert countgraph.n_occupied() != 0
     hashsizes = countgraph.hashsizes()
     kmer_size = countgraph.ksize()
-    tracking = khmer._Nodegraph(kmer_size, hashsizes)
+    tracking = khmer.Nodegraph(kmer_size, 1, 1, primes=hashsizes)
```

Comment:
Almost all of the changes in this file are related to removal of underscore-prefixed constructors.